src/music/element/MusicQueue.py

```diff
@@ -9,8 +9,11 @@
 class MusicQueue:
     """Reperesents a Queue GUI object. Handles which MusicSource to play next and displays in the GUI.
     """
-    def __init__(self, active_embed: ChatEmbed, client: discord.Client, list: List[MusicSource] = []) -> None:
-        self.list = list
+    def __init__(self, active_embed: ChatEmbed, client: discord.Client, playlist: List[MusicSource] = None) -> None:
```

Comment:
Any reason why you had to set playlist to None instead in the arguments instead of just having it set as an empty list by default